core/Cargo.toml

```diff
@@ -27,13 +27,15 @@ color-backtrace = { version = "0.3", optional = true, default-features = false }
 generational-arena = { version = "0.2", optional = true }
 gumdrop = { version = "0.7", optional = true }
 lazy_static = "1"
-log = { version = "0.4", optional = true, features = ["std"] }
 regex = { version = "1", optional = true }
 secrecy = { version = "0.5", optional = true, features = ["serde"] }
 semver = { version = "0.9", optional = true }
 serde = { version = "1", optional = true, features = ["serde_derive"] }
 termcolor = { version = "1", optional = true }
 toml = { version = "0.5", optional = true }
+tracing = { version = "0.1", optional = true }
+tracing-log = { version = "0.1", optional = true }
+tracing-subscriber = { version = "0.1", optional = true }
```

Comment:
There are a number of feature flags this crate exposes which should probably be turned on or off based on what is being used.

Reply:
Would definitely be a fan of shutting off any unused features, or at least making them optional.

Reply:
In particular, the `smallvec` and `parking_lot` optional dependencies are there only to allow users to switch them on if benchmarking indicates that they provide performance benefits in a particular application; it might be worth exposing these to the user to turn on if needed?